packages/network_traffic/data_stream/amqp/fields/beats.yml

```
@@ -54,10 +54,35 @@
     VLAN identifier from the 802.1q frame. In case of a multi-tagged frame this field will be an array with the outer tag's VLAN identifier listed first.
 
 - name: type
+  type: keyword
   description: >
     The type of the transaction (for example, HTTP, MySQL, Redis, or RUM) or "flow" in case of flows.
 
+- name: process.name
+  type: keyword
+  description: >
+    The name of the process.
```

ECS defines these process fields so I would like to see it import those definitions.

```suggestion
- name: process.name
  external: ecs
```